prime-angular-frontend/src/app/shared/components/banner-wrapper/banner-wrapper.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { BannerLocationCode } from '@shared/enums/banner-location-code.enum';
+import { Banner } from '@shared/models/banner.model';
+import { BannerResourceService } from '@shared/services/banner-resource.service';
+
+@Component({
+  selector: 'app-banner-wrapper',
+  templateUrl: './banner-wrapper.component.html',
+  styleUrls: ['./banner-wrapper.component.scss']
+})
+export class BannerWrapperComponent implements OnInit {
```

Comment:
Feels like we don't need the BannerWrapperComponent.  The BannerComponent can be placed on a page and passed a location code, be internally responsive, get the data it requires, and display an alert without being wrapped.  The location code can also be passed in through the route data snapshot, and if it doesn't exist then internally the banner hides it's content.  So all you would need to do is include `<app-banner></app-banner>` on a page and it self manages.